src/components/MovieList/MovieList.jsx

```diff
@@ -1,3 +1,16 @@
 import './MovieList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MovieList = () => <>Put the list here</>;
+export function MovieList({ movies }) {
+  if (!movies || movies.length === 0) {
+    return null;
+  }
```

Comment:
If you don't have `movies` you don't need to render `MovieList` component at all